### ISSUE TITLE
Feat: Riemannian Optimization Alternative

### ISSUE THREAD

**Issue by zazabap** [nzy1997/ParametricDFT.jl#29]
Give other options for the optimizer. Riemannian Gradient Descent might not be so effective after visualization of the training loss. Shown in PR #27  Considering other options for the optimizer.
